### ISSUE TITLE
Unable to download CSV in Microsoft Edge

### ISSUE THREAD

**Issue by eguitarz** [highcharts/export-csv#76]
When I was exporting csv under Microsft Edge browser in export-csv example page (http://jsfiddle.net/highcharts/cqjvD/), the browser crashed and not able to download CSV file.

**Comment by TorsteinHonsi** [comment]
@oysteinmoseng Will you have a look at this? My Win10 machine is crashing.

**Comment by oysteinmoseng** [comment]
@TorsteinHonsi Would love to, but am still on Win7... Is @jon-a-nygaard on Win10 yet?

**Comment by amsterdamharu** [comment]
I fixed this changing the following in export-csv.js:

        if (downloadAttrSupported) {
            a = document.createElement('a');
            a.href = href;
            a.target      = '_blank';
            a.download    = name + '.' + extension;
            document.body.appendChild(a);
            a.click();
            a.remove();

        } else if (window.Blob && window.navigator.msSaveOrOpenBlob) {
            // Falls to msSaveOrOpenBlob if download attribute is not supported
            blobObject = new Blob([content]);
            window.navigator.msSaveOrOpenBlob(blobObject, name + '.' + extension);

        }...

To:

        if (window.Blob && window.navigator.msSaveOrOpenBlob) {
            // Falls to msSaveOrOpenBlob if download attribute is not supported
            blobObject = new Blob([content]);
            window.navigator.msSaveOrOpenBlob(blobObject, name + '.' + extension);

        } else if (downloadAttrSupported) {
            a = document.createElement('a');
            a.href = href;
            a.target = '_blank';
            a.download = name + '.' + extension;
            document.body.appendChild(a);
            a.click();
            a.remove();

        } ....

**Comment by TorsteinHonsi** [comment]
I am running Edge now, but actually I was able to download the CSV without a problem...

**Comment by jon-a-nygaard** [comment]
@oysteinmoseng I am also still on Win7

**Comment by oysteinmoseng** [comment]
Does @amsterdamharu's fix work? Seems to make sense

**Comment by TorsteinHonsi** [comment]
For me it works even without the fix. In theory, it makes more sense first to test for the W3C way of doing things then fall back to the proprietary feature, but as you know, pragmatism trumps perfection.

**Comment by asafpelegcodes** [comment]
I just ran across the an issue on Microsoft Edge (v 25.10586.0.0) running in windows 10.  

When I tested the export csv I debugged my application and it's going into the following branch.
```
if (downloadAttrSupported) {
        a = document.createElement('a');
        a.href = href;
        a.target      = '_blank';
        a.download    = name + '.' + extension;
        document.body.appendChild(a);
        a.click();
        a.remove();

    }
```
and all that happens is another blank tab opens, no download occurs.

Just for experimentation I manually edited the file with the suggestion that @amsterdamharu posted above and it fixed the problem suggesting there is an issue with the branch I pasted above.
